ConceptMap/ConceptMap-Genomics-primary-sample-state.json

```diff
@@ -0,0 +1,141 @@
+    {
+      "name": "NHS England",
+      "telecom": [
+        {
+          "system": "email",
+          "value": "interoperabilityteam@nhs.net"
+        }
+      ]
+    }
+  ],
+  "description": "A ConceptMap used for the translation and recording of concepts from MDS source data(code-options) to equivalent Snomed CT codes, and FHIR spec.",
+  "purpose": "ConceptMap used for the translation and recording the concepts on method of sample preservation. This is intended for the element Specimen.condition",
+  "copyright": "Copyright © 2023+ NHS England Licensed under the Apache License, Version 2.0 (the \\\"License\\\"); you may not use this file except in compliance with the License. You may obtain a copy of the License at  http://www.apache.org/licenses/LICENSE-2.0 Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an \\\"AS IS\\\" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License. HL7® FHIR® standard Copyright © 2011+ HL7 The HL7® FHIR® standard is used under the FHIR license. You may obtain a copy of the FHIR license at  https://www.hl7.org/fhir/license.html.",
+  "targetUri": "http://snomed.info/sct?fhir_vs=ecl/%3C%20260787004%20",
+  "group": [
+    {
+      "target": "http://snomed.info/sct",
+      "element": [
+        {
+          "code": "DNA-stabiliser",
+          "display": "DNA Stabiliser",
+          "target": [
+            {
+              "code": "706052002",
+              "display": "Evacuated blood collection tube (physical object)",
+              "equivalence": "equivalent"
+            }
+          ]
+        },
+        {
+          "code": "edta",
+          "display": "EDTA",
+          "target": [
+            {
+              "code": "702282003",
+              "display": "Non-evacuated blood collection tube, EDTA",
+              "equivalence": "equivalent"
+            }
+          ]
+        },
+        {
+          "code": "ffpe-curls/scrolls",
+          "display": "FFPE Curls/Scrolls",
+          "target": [
+            {
+              "equivalence": "unmatched"
+            }
+          ]
+        },
+        {
+          "code": "ffpe-slides",
+          "display": "FFPE Slides",
+          "target": [
+            {
+              "equivalence": "unmatched"
+            }
+          ]
+        },
+        {
+          "code": "fresh-cells",
+          "display": "Fresh Cells",
+          "target": [
+            {
+              "code": "706049005",
+              "display": "Blood tube (physical object)",
+              "equivalence": "subsumes"
+            }
+          ]
+        },
+        {
+          "code": "fresh-tissue",
+          "display": "Fresh Tissue",
+          "target": [
+            {
+              "code": "434464009",
+              "display": "Tissue cassette (physical object)",
```

Comment:
441479001 | Fresh tissue specimen (specimen) | seems more appropriate, though if we are coding container types, the concept type does not match